include/repeater_cadmium.hpp

```diff
@@ -0,0 +1,199 @@
+/** \brief This header file implements the Repeater class
+ * that acts as a module that takes input from subnet and
+ * after processing it send it to the receiver and get
+ * acknowledgement from receiver and forward it to sender
+ * after processing it.
+ *
+ * Repeater has 2 inputs and 2 outputs port for packet and
+ * acknowledgement respectively. It adds a 10 second
+ * processing time to packets and send it to respective output ports
+*/
+/*
+ * Peter Bliznyuk-Kvitko
+ * Shubham Agrawal
+ * Carleton University
+*/
+
+#ifndef __REPEATER_CADMIUM_H__
+#define __REPEATER_CADMIUM_H__
+
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/message_bag.hpp>
+#include <limits>
+#include <math.h>
+#include <assert.h>
+#include <memory>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <chrono>
+#include <algorithm>
+#include <limits>
+#include <random>
+
+#include "message.hpp"
+
+using namespace cadmium;
+using namespace std;
+/**
+ * Structure that holds input and output messages
+ * using input and output port.
+*/
+struct repeater_defs {
+    struct packet_sent_out : public out_port<Message_t> {
+    };
+    struct ack_received_out : public out_port<Message_t> {
+    };
+    struct packet_in : public in_port<Message_t> {
+    };
+    struct ack_in : public in_port<Message_t> {
+    };
+};
+
+/**
+ * The Sender class sends out messages and receives acknowledges.
```

Comment:
I could see that the comments are misaligned in this file. Please see the attached screenshot. 
![image](https://user-images.githubusercontent.com/44588557/63488016-6bcb0700-c47b-11e9-8a6e-b8d15ffa8634.png)
Please check the entire file. I see at other places as well (for example 172, 190).

Reply:
Also, ensure that the comments doesn't exceed 80 characters.

Reply:
In line 59, I would suggest to use the same formatting style (although the comment is just one line).

Reply:
fixed can you please check now

Reply:
looks good now. the entire has the same commenting style